tools/misbehaving-jmx-server/pom.xml

```diff
@@ -0,0 +1,127 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>org.datadog</groupId>
+  <artifactId>misbehavingjmxserver</artifactId>
+  <version>1.0-SNAPSHOT</version>
+
+  <name>misbehavingjmxserver</name>
+  <!-- FIXME change it to the project's website -->
+  <url>http://www.datadoghq.com</url>
+
+  <properties>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    <java.version>11</java.version>
```

Comment:
Currently this project requires java 11 largely due to the `javalin` dependency. While `javalin` provides a very nice interface for an http server, it does limit our options if we want to run this test application in an older JVM.
But maybe we never need to test an older JVM, java 11 came out in 2018 and is the current LTS version.